modules/lobby/src/main/SeekApi.scala

```diff
@@ -59,7 +59,9 @@ final class SeekApi(
       .foldLeft(List.empty[Seek] -> Set.empty[String]) {
         case ((res, h), seek) if seek.user.id == user.id => (seek :: res, h)
         case ((res, h), seek) =>
-          val seekH = List(seek.variant, seek.daysPerTurn, seek.mode, seek.playerIndex, seek.user.id) mkString ","
+          val seekH =
+            List(Some(seek.variant.toString()), seek.daysPerTurn.map(_.toString()), Some(seek.mode.toString()), Some(seek.playerIndex), Some(seek.user.id))
```

Comment:
This seems unnecessary, why all the Some()?

Reply:
daysPerTurn is an Option, but the rest are not. I had to convert to string to keep compiler happy, and keep objects inside list all the same type. If dayPerTurn is None then i dont want it in the list. 
I could have used fold but that seems like more lines of code e.g. 
```
seek.daysPerTurn.fold(List(seek.variant.toString(), seek.mode.toString(), seek.playerIndex, seek.user.id))(List(seek.variant.toString(), _.toString(), seek.mode.toString(), seek.playerIndex, seek.user.id)).mkString(",")
```